### DISCUSSION TITLE
Time for a New Version of Virt-manager (4.2.0)

### DISCUSSION THREAD

**Discussion by charlesa** [virt-manager/virt-manager#712] [General]
It has been two years since virt-manager 4.1.0 was tag'ed. Since that time, there have been approximately 271 commits . May the tree be tagged with 4.2.0 and a source tarball generated?

**Comment by crobinso** [comment]
Yes we are way way overdue. I was hoping to get a release out before my paternity leave over the summer, but that didn't happen. I'm back to speed now and devoting my near term time to virt-manager. I will do my best to get a release out by end of the month.

**Reply by charlesa** [reply]
Thanks for the update and congratulations on your new baby!

**Comment by phrdina** [comment]
Hi, new version 5.0.0 was released last week so closing this discussion.
